k6/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "121-service-k6-tests",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "lint": "eslint .",
+    "fix": "npm run lint -- --fix",
+    "test": "dotenv -e ../services/.env -- k6 run"
```

Comment:
I left this line here by mistake but now I'm wondering if it actually works.

You could try it by running something like:

```shell
npm run test -- tests/pvProgramPerformance.js
```

If that works, then you should update the README accordingly. If not, remove this line.

Reply:
If it was by mistake then make more of those. I'll update readme. I tested it locally.